frontend/src/component/common/InstanceStatus/InstanceStatus.tsx

```diff
@@ -104,7 +104,7 @@ export const InstanceStatus: FC = ({ children }) => {
     return (
         <>
             <ConditionallyRender
-                condition={isBilling && Boolean(instanceStatus)}
+                condition={Boolean(instanceStatus)}
```

Comment:
If I understood this correctly, we want `isBilling` to only represent plans that we show the billing page for (invoices, etc). That's why we removed it from this condition and have not added ENTERPRISE to the billing plans.

I think this is a safe change, both the bar and the dialog should only get displayed in case we have specific information from the instance status (e.g. has expired).

Reply:
> I would recommend removing isBilling from the destructor above since it's now unused.

Yes!